pkgs/tools/backup/hpe-ltfs/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, fuse, icu, pkgconfig, libxml2, libuuid }:
+
+stdenv.mkDerivation rec {
+  version = "3.4.2_Z7550-02501";
+  pname = "hpe-ltfs";
+
+  src = fetchurl {
+    url = "https://github.com/nix-community/hpe-ltfs/archive/${version}.tar.gz";
+    sha256 = "0jwqpjlv92ld6v4ay3qjsjk94xq6gmk0kwdwz12bkaqaygymrcj7";
+  };
```

Comment:
I believe we should use `fetchFromGitHub` here.
It uses `fetchzip` which hashes the content.